docker/install/ubuntu_init_zephyr_project.sh

```diff
@@ -29,20 +29,26 @@ set -x
 
 DOWNLOAD_DIR=$1
 shift
-ZEPHYR_BRANCH=$1
-shift
 
-commit_hash=
+if [ "$1" == "--zephyr-branch" ]; then
+    shift
+    ZEPHYR_BRANCH=$1
+else
+    ZEPHYR_BRANCH="v2.7-branch"
+    shift
```

Comment:
I'm not sure this second `shift` here is needed, as in case `$1` is not `--zephyr-branch`, it will be incorrectly skipped?

Reply:
yes, thanks for catching. It should be fixed now.